packages/components/src/box-control/utils.ts

```diff
@@ -26,16 +27,16 @@ export const DEFAULT_VALUES = {
 	left: undefined,
 };
 
-export const ALL_SIDES = [ 'top', 'right', 'bottom', 'left' ];
+export const ALL_SIDES = [ 'top', 'right', 'bottom', 'left' ] as const;
 
 /**
  * Gets an items with the most occurrence within an array
  * https://stackoverflow.com/a/20762713
  *
- * @param {Array<any>} arr Array of items to check.
- * @return {any} The item with the most occurrences.
+ * @param arr Array of items to check.
```

Comment:
Rady @aaronrobertshaw 's earlier reviews, could it be that our linters have slightly different outputs? 🤔

Reply:
Mmm, still the same after an `npm ci` with trunk and `npm run lint:js -- packages/components/src/box-control/utils.ts` 🤔 No `box-control/` [warnings on CI](https://github.com/WordPress/gutenberg/actions/runs/4075613203/jobs/7022242451) either. In any case there are a lot of existing alignment warnings, and since they are only warnings, I expect they will continue to flip-flop for a while 😅